o2a/templates/mapreduce.tpl

```diff
@@ -28,10 +28,8 @@
         archives={{ hdfs_archives | to_python }},
     {% endif %}
     cluster_name=CONFIG['dataproc_cluster'],
-    dataproc_hadoop_properties={% include "props.tpl" %},
-    dataproc_hadoop_jars=CONFIG['hadoop_jars'].split(','),
+    dataproc_properties={% include "props.tpl" %},
```

Comment:
I think this change shouldn't be here. First of all it doesn't pertain to the PR's aim (Java mapper), and secondly - isn't this a "non-issue" that we discussed [here](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/294)?

Reply:
ah. Good Catch. It was not supposed to be there. I wonder how it came to be here in the first place.

Reply:
Fixed in dcf31ed